packages/image/src/image-optimize.ts

```diff
@@ -131,6 +130,19 @@ const getWidths = (
     return { widths: deviceSizes, kind: "w" };
   }
 
+  const MAX_DEVICE_PIXEL_RATIO = 2;
```

Comment:
is this running serverside? in browser there is window.devicePixelRatio

Reply:
Yep serverside so we use max. This is for srcSet generation so we don't need real pixel-ratio, just max - which is 2 (in real can be 3,4 etc but for images there is not need > 2)

Reply:
good info, added as a comment

Reply:
done